ci-operator/step-registry/redhat-chaos/hog-scenarios/io/OWNERS

```diff
@@ -0,0 +1,10 @@
+approvers:
```

Comment:
This file should reference the OWNERS file in the main redhat-chaos folder, this way we only need to update one file for the whole repo and it'll stay up to date

you can create a symbolic link using
ln -s ./OWNERS ./<folder_name>